src/wrap_cl_part_1.cpp

```diff
@@ -309,6 +309,17 @@ void pyopencl_expose_part_1(py::module_ &m)
       py::arg("wait_for").none(true)=py::none()
       );
 
+#ifdef CL_DEVICE_P2P_DEVICES_AMD
+  m.def("enqueue_copy_buffer_p2p_amd", enqueue_copy_buffer_p2p_amd,
```

Comment:
Please add to the documentation, including `.. versionadded`.

Reply:
Wasn't quite sure what to put for versionadded, so I just put `2023.8`